composer.json

```diff
@@ -10,9 +10,16 @@
         }
     ],
     "minimum-stability": "dev",
+    "repositories":
+    [
+            {
+                "type": "vcs",
+                "url": "https://github.com/arborrow/laravel-phone-validator"
+            }
+    ],
```

Comment:
Im guessing this isn't supposed to be here?

Reply:
Correct, I should have branched before I continued playing with my local branch